en/frames_multicopter/dji_f450_cuav_5plus.md

```diff
@@ -0,0 +1,212 @@
+
+Estimated time to assemble is approximately 90 minutes (about 45 minutes for the frame and 45 minutes installing the autopilot and configuring the airframe.
+
+1. Attach the 4 arms to the bottom plate using the provided screws.
+
+   ![Arms to bottom plate](../../assets/airframes/multicopter/dji_f450_cuav_5plus/1_attach_arms_bottom_plate.jpg)
+
+1. Solder ESC (Electronic Speed Controller) to the board, positive (red) and negative (black).
+
+   ![Solder ESCs](../../assets/airframes/multicopter/dji_f450_cuav_5plus/2_solder_esc.jpg)
+
+1. Solder the Power Module, positive (red) and negative (black).
+
+   ![Solder power module](../../assets/airframes/multicopter/dji_f450_cuav_5plus/3_solder_power_module.jpg)
+
+1. Plug in the motors to the ESCs according to their positions.
+
+   ![Plug in motors](../../assets/airframes/multicopter/dji_f450_cuav_5plus/4_plug_in_motors.jpg)
+
+1. Attach the motors to the corresponding arms.
+
+   ![Attach motors to arms (white)](../../assets/airframes/multicopter/dji_f450_cuav_5plus/5a_attach_motors_to_arms.jpg)
+   ![Attach motors to arms (red)](../../assets/airframes/multicopter/dji_f450_cuav_5plus/5b_attach_motors_to_arms.jpg)
+
+1. Add the top board (screw into the top of the legs).
+
+   ![Add top board](../../assets/airframes/multicopter/dji_f450_cuav_5plus/6_add_top_board.jpg)
+
+1. Add double-sided tape to the CUAV V5+ flight controller (it has internal vibration damping, so no need to use foam).
+
+   ![Damping foam](../../assets/airframes/multicopter/dji_f450_cuav_5plus/7_attach_cuav5plus.jpg)
+
+1. Attach the FrSky receiver to the bottom board with double-sided tape.
+
+   ![Attach FrSky receiver with double-sided tape](../../assets/airframes/multicopter/dji_f450_cuav_5plus/8_attach_frsky.jpg)
+
+1. Attach the telemetry module to the vehicle’s bottom board using double-sided tape.
+
+   ![Attach telemetry radio](../../assets/airframes/multicopter/dji_f450_cuav_5plus/9a_telemtry_radio.jpg)
+   ![Attach telemetry radio](../../assets/airframes/multicopter/dji_f450_cuav_5plus/9b_telemtry_radio.jpg)
+
+1. Put the aluminium standoffs on the bottom plate.
+
+1. Plug in Telemetry (`TELEM`) and GPS module (`GPS/SAFETY`) to the flight controller.
+   ![Attach GPS](../../assets/airframes/multicopter/dji_f450_cuav_5plus/11a_gps.jpg)
+   ![Attach GPS](../../assets/airframes/multicopter/dji_f450_cuav_5plus/11b_gps.jpg)
+   
+1. Plug in the RC receiver (`RC`), all 4 ESC’s (`M1-M4`), and the power module (`Power1`) into the flight controller.
+   ![Attach peripherals to flight controller](../../assets/airframes/multicopter/dji_f450_cuav_5plus/12_fc_attach_periperhals.jpg)
+   
+   > **Note** The motor order is defined in the [Airframe Reference > Quadrotor x](../airframes/airframe_reference.md#quadrotor-x)
+
+That's it!
+The final build is shown below:
+
+![Finished Setup](../../assets/airframes/multicopter/dji_f450_cuav_5plus/f450_cuav5_plus_complete_2.jpg)
+
+
+## Vehicle Configuration/Calibration {#configure}
```

Comment:
@Tony3dr  FYI, I've pretty much standardised on the setup for config of vehicles with this section. It is always pretty much the same, because config should all be much same. The exception is that I add image for the airframe selection, and of course we would add more if there was any extra equipment